ACCEPTANCE.md

```
@@ -0,0 +1,16 @@
+## Definition of Done — E2E Regression Guard
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)